samples/Azure.Management.Storage/Generated/Models/BlobInventoryPolicyFilter.Serialization.cs

```diff
@@ -51,6 +51,10 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
 
         internal static BlobInventoryPolicyFilter DeserializeBlobInventoryPolicyFilter(JsonElement element)
         {
+            if (element.ValueKind == JsonValueKind.Null)
```

Comment:
It appears we have 2 pattern changes in here?  I would like to keep this uber large regens to a single pattern if possible as it makes it much easier to review.  Lets keep this one focused on the null element in dictionary / list and we can add this in a separate PR.

Reply:
Right. The simplest solution is for every `JsonSerialization.IsNullable : true`, add null check in serialization/deserialization codes. The side effect is we'll have a lot unnecessary null check.

- `string`: both `Utf8JsonWriterExtensions.WriteStringValue()` and `JsonElement.GetString()` can handle null value correctly
- `byte[]`: we can modify it's serialization/deserialization method to handle null value correctly
- custom models: add the null value handling in deserialization method, which could also benefit for future work if we want to support nullable optional properties

For the 1st stage, we could roll back to the simplest solution.